opacus/privacy_engine.py

```diff
@@ -53,8 +53,11 @@ def __init__(self, accountant: str = "rdp", secure_mode=False):
 
         """
         self.accountant = get_accountant(mechanism=accountant)
+        self.accounting_mechanism = accountant
```

Comment:
Suggestion: we can have an attribute in accountant and then access it as `self.accountant.mechanism`